libs/application/templates/general-fishing-license/src/forms/GeneralFishingLicensePaymentForm.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  buildForm,
+  buildSection,
+  Form,
+  FormModes,
+  buildCustomField,
+} from '@island.is/application/core'
+import {
+  applicantInformation,
+  conclusion,
+  externalData,
+  fishingLicense,
+  overview,
+  payment,
+  shipSelection,
+} from '../lib/messages'
+
+export const GeneralFishingLicensePaymentForm: Form = buildForm({
+  id: 'GeneralFishingLicensePaymentForm',
+  title: '',
+  mode: FormModes.APPLYING,
+  renderLastScreenButton: false,
+  children: [
+    buildSection({
+      id: 'ExternalDataSection',
+      title: externalData.dataProvider.sectionTitle,
+      children: [],
```

Comment:
Why are there so many empty sections? Are you doing this to fake the progress of the status bar?

Reply:
Yes, this is to fake the status bar, it is done in all the applications that use payment